### PR TITLE
Navbar_rework

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

File: src/components/app/components/Navbar.js [hunk 1]
```diff
@@ -73,21 +67,30 @@ class Navbar extends React.Component {
...
-                        })
-                    }
+                    <Item key={AVATAR}>
+                        <AppLink key={AVATAR} to={HOME}>
+                            <Avatar className={AVATAR} src={SOURCE} size={SIZE} shape={SHAPE} />
+                        </AppLink>
+                    </Item>
+                    <Item key={MARKET}>
+                        <AppLink key={MARKET} to={MARKET}>{MARKET_TEXT}</AppLink>
+                    </Item>
```
Comment (Way2nnadi) [inline src/components/app/components/Navbar.js]: Why did you remove the `items.map` here?

Comment (Simpcyclassy) [reply src/components/app/components/Navbar.js]: At a point my code ran into error I decided to split. I was going to put that back
